src/shared/converters/image.js

```diff
@@ -40,45 +39,67 @@ export default {
           eventAction="open content media"
         >
           <a>
-            <Image content key={attachmentId} id={attachmentId} />
+            <Image isContent key={attachmentId} id={attachmentId} />
           </a>
         </Link>
       );
     }
 
-    let src;
+    const alt = attributes.alt || null;
 
-    // Get src attribute from different cases or assign an empty string.
+    let src = null;
+
+    // Get src attribute from different cases.
     if (attributes.src && typeof attributes.src === 'string') {
-      ({ src } = attributes);
+      if (attributes.src.startsWith('http')) src = he.decode(attributes.src);
+      else src = he.decode(`${settings.generalSite.url}${attributes.src}`);
     } else if (
       attributes.dataset &&
       attributes.dataset.original &&
       typeof attributes.dataset.original === 'string'
     ) {
-      src = attributes.dataset.original;
-    } else {
-      src = '';
+      if (attributes.src.startsWith('http'))
+        src = he.decode(attributes.dataset.original);
+      else
+        src = he.decode(
+          `${settings.generalSite.url}${attributes.dataset.original}`,
```

Comment:
Same here. We should check if it starts with `/` instead of `http`.

By the way, what's the purpose of `data-original`?

Reply:
@luisherranz Somehow the src can come in `data-original` as well. Don't remember the particular case but it happened before.

Reply:
Hmm.. ok, I don't remember the case either. Maybe @DAreRodz does?